File: 3110-ScoreOfAString/3110-ScoreOfAString.cpp
// Last updated: 9/10/2026, 9:47:02 AM
class Solution {
public:
    int scoreOfString(string s) {
        int sum=0;
        for(int i=0;i<s.size()-1;i++){
            sum+=(abs((s[i]-s[i+1])));
        }
        return sum;
    }
};

File: 3110-ScoreOfAString/3110-ScoreOfAString.py
# Last updated: 9/10/2026, 9:47:08 AM
class Solution(object):
    def scoreOfString(self, s):
        """
        :type s: str
        :rtype: int
        """
        sum=0
        for i in range(len(s)-1):
            sum=sum+(abs(ord(s[i])-ord(s[i+1])))
        return sum

        